app/services/waste_exemptions_engine/refresh_companies_house_name_service.rb

```diff
@@ -7,6 +7,8 @@ class RefreshCompaniesHouseNameService < WasteExemptionsEngine::BaseService
     def run(company_reference)
       registration = Registration.find_by(reference: company_reference)
       company_name = DefraRubyCompaniesHouse.new(registration.company_no).company_name
+      raise StandardError if company_name == nil?
```

Comment:
Use `.new` and give the exception a message, e.g: `raise StandardError.new "Company name not found"`, or just `raise "Company name not found"`